### ISSUE TITLE
Module specific flow

### ISSUE THREAD

**Issue by IonicaBizau** [jillix/flow-nodejs#138]
Is it possible to have module specific flow configuration that will be interpreted at module level, without needing to add it in the application composition?

**Comment by ottiker** [comment]
Yes, it's possible to create flow event handlers and append it to whatever observer you want.
Here's an example to use flow configurations on the module level: [adioo/view/view.js#L261-L273](https://github.com/adioo/view/blob/master/view.js#L261-L273)
And here's is how engine use it internally to setup the event flow: [jillix/engine/client/engine.js#L676](https://github.com/jillix/engine/blob/v0.1.0-beta/lib/client/engine.js#L676)

Basically `engine.flow(module_instance, flowConfig[i].out)` creates the event handler, which you can append to the `flowConfig[i].in` event name.

**Comment by IonicaBizau** [comment]
Wow! It's working. Thanks!
